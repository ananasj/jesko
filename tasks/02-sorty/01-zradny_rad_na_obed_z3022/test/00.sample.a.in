7
Kaja
Luxusko
Kata
Baska
Maja
Sysel
Olivia
