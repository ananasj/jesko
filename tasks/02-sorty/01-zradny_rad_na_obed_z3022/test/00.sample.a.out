o 2 dozadu
